src/store/modules/routingSlip.ts

```diff
@@ -150,6 +150,32 @@ export default class RoutingSlipModule extends VuexModule {
     return this.routingSlip?.status === SlipStatus.VOID
   }
 
+  @Mutation
+  public updateRoutingSlipChequeNumber (chequeNumToChange) {
```

Comment:
what is the type of chequeNumToChange?

Reply:
it's object, I add an interface of AdjustRoutingSlipChequePrams for the type